developer/src/kmldmlc/src/keyman/kmx/kmx-builder.ts

```diff
@@ -195,20 +195,30 @@ export default class KMXBuilder {
       groupBase += KMXFile.COMP_GROUP_SIZE;
     }
 
-    size += this.calculateBitmapSize();
-    size += this.calculateKMXPlusSize();
+    size += this.buildBitmap();
+    size += this.buildKMXPlus(size);
 
     return size;
   }
 
-  calculateBitmapSize() {
+  buildBitmap() {
     // TODO
     return 0;
   }
 
-  calculateKMXPlusSize() {
-    // TODO
-    return 0;
+  buildKMXPlus(base: number) {
```

Comment:
So both buildBitMap() and buidlKMXPlus() return a size?

```suggestion
  buildKMXPlus(base: number): number {
```

Reply:
Yes, but we don't need to explicitly set the output type because Typescript can infer it.